build/build-modules-js/settings.json

```diff
@@ -85,6 +85,14 @@
               "defer": true
             }
           },
+          {
```

Comment:
@Krshivam25 per glip, this is in the wrong place and likely to be causing the mousetrap npm error.

- This has been defined inside the bootstrap settings, which is incorrect.
- Move lines 88 to 95 down to near line 543 where punycode is called (to keep it alphabetical)
- Add `"mousetrap": ` before the starting bracket to define it's the settings for the mousetrap library.

Reply:
And after doing all of that take a look at everything else and you will see that you have all the options wrong

![image](https://user-images.githubusercontent.com/1296369/141963415-112f603b-2d36-44af-806d-c1d9f35de890.png)